nfc/NfcCx/Cx/NfcCxSC.cpp

```diff
@@ -1718,6 +1722,42 @@ Return Value:
 
         cbOutputBufferUsed = cbResponseBuffer;
     }
+    else if (NfcCxSCInterfaceValidateSwitchProtocolCommand((PBYTE)InputBuffer + sizeof(SCARD_IO_REQUEST),
+        cbInputBuffer - sizeof(SCARD_IO_REQUEST)) == TRUE){
+        BYTE Sw1Sw2[DEFAULT_APDU_STATUS_SIZE] = { 0x90,0x00 };
+
+        status = NfcCxSCInterfaceDeactivateMultiProtocolTag(scInterface);
+        if (NT_SUCCESS(status)){
+            WdfWaitLockAcquire(scInterface->SmartCardLock, NULL);
+
+            status = NfcCxSCInterfaceLoadSelectedProtocol(
+                                                        scInterface,
+                                                        (PBYTE)InputBuffer + sizeof(SCARD_IO_REQUEST),
+                                                        cbInputBuffer - sizeof(SCARD_IO_REQUEST));
+
```

Comment:
Failure logging & code comments has be added.
Here  irrespective of status of this function, next sequence (NfcCxSCInterfaceActivateMultiProtocolTag) is needed as in previous sequence (NfcCxSCInterfaceDeactivateMultiProtocolTag) tag deactivation was done.